### PR TITLE
Handle comment conflicts

### PR DESCRIPTION
**Pull request by atelic**

### CODE REVIEW

File: website/static/js/registrationUtils.js [hunk 1]
```diff
@@ -44,6 +45,8 @@ function Comment(data) {
         self.lastModified = new Date();
     });
 
+    self.created = data.created || new Date();
```
Comment (samchrisinger) [inline website/static/js/registrationUtils.js:48]: @barbour-em should we be parsing this into a Date object?

Comment (atelic) [inline website/static/js/registrationUtils.js:48]: @samchrisinger what do you mean? Should `data.created` become `new Date(data.created)`?

Comment (samchrisinger) [inline website/static/js/registrationUtils.js:48]: Yes. 

I've already merged this code, but wondering if I should fix this

Comment (atelic) [inline website/static/js/registrationUtils.js:48]: Yes, that likely should be the case.

File: website/project/metadata/utils.py [hunk 2]
```diff
@@ -21,8 +21,8 @@ def serialize_draft_registration(draft, auth=None):
         'registration_schema': serialize_meta_schema(draft.registration_schema),
         'initiated': str(draft.datetime_initiated),
         'updated': str(draft.datetime_updated),
-        'config': draft.config or {},
-        'flags': draft.flags,
+        'config': {}, # or draft.config or {},
```
Comment (samchrisinger) [inline website/project/metadata/utils.py:24]: @barbour-em as a note, this formatting is a flake8 exception. Two spaces should proceed all inline comments.

**Comment by samchrisinger** [comment]
Merged manually.

**Pull request closed without merging**